gluten-ut/spark32/src/test/scala/org/apache/gluten/utils/velox/VeloxTestSettings.scala

```diff
@@ -364,6 +364,8 @@ class VeloxTestSettings extends BackendTestSettings {
     // Need to support MAP<NullType, NullType>
     .exclude(
       "SPARK-27619: When spark.sql.legacy.allowHashOnMapType is true, hash can be used on Maptype")
+    .exclude("SPARK-17515: CollectLimit.execute() should perform per-partition limits")
```

Comment:
Can you add the exclude reason here?

Reply:
I believe there were some benign failures which can be solved in another time.

Reply:
Yes, can you add the detail error message upon this exclude line, like `// Need to support MAP<NullType, NullType>`

Reply:
Let's get this merged so you can proceed with https://github.com/apache/incubator-gluten/pull/8914.

One can blame the change to find this PR for failure details. The error log is [here](https://github.com/apache/incubator-gluten/actions/runs/14089831855/job/39463604913).

Just revert the test changes once implemented a new rule following https://github.com/apache/incubator-gluten/pull/9145#discussion_r2016227274.